exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
+let greetingStart = "Hello, my name is ";
+let name = "Musa";
+let message = greetingStart + name;
 console.log(message);
```

Comment:
Good Musa, now as a stretch I want you to show me how you would console log that same message in string literal syntax instead of string conjugation. Here's the docs for it: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals